packages/clerk-js/src/ui/components/SubscriptionDetails/index.tsx

```
@@ -270,7 +276,10 @@ const SubscriptionDetailsFooter = withCardStateProvider(() => {
 });
 
 function SubscriptionDetailsSummary() {
-  const { anySubscription, activeSubscription, upcomingSubscription } = useGuessableSubscription({ or: 'throw' });
+  const { anySubscription, activeSubscription, upcomingSubscription } = useGuessableSubscription({
+    or: 'throw',
+  });
+  const { t } = useLocalizations();
```

ESLint

[error] 't' is assigned a value but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)